.github/workflows/storybook-check.yml

```
@@ -34,14 +34,14 @@ jobs:
             - name: Build Storybook
               run: npm run storybook:build
 
-            # - name: Install Playwright dependencies
-            #   run: npx playwright install --with-deps
-
-            # - name: Serve Storybook and run tests
-            #   run: |
-            #       npx concurrently -k -s first -n "SB,TEST" -c "magenta,blue" \
-            #       "npx http-server ./storybook/build --port 50240 --silent" \
-            #       "npx wait-on tcp:127.0.0.1:50240 && \
-            #       NODE_PATH=./node_modules \
-            #       npx --package=@storybook/test-runner -- \
-            #       test-storybook --url http://localhost:50240 --config-dir ./storybook"
+            - name: Install Playwright dependencies
+              run: npx --no-install playwright install --with-deps
```

Should this be `--no` instead of `--no-install` ? From the [documentation](https://docs.npmjs.com/cli/v8/commands/npx), this appears to be a deprecated option.

>The `--no-install` option is deprecated, and will be converted to `--no`.

```suggestion
              run: npx --no playwright install --with-deps
```